framework/Samplers/Sampler.py

```diff
@@ -640,6 +617,61 @@ def localStillReady(self,ready): #,lastOutput=None
     """
     return ready
 
+  def _checkRestartForEvaluation(self):
+    """
+      Checks restart data object (if any) for matching realization.
+      @ In, None
+      @ Out, index, int, index of matching realization in restart (None if not found)
+      @ Out, inExisting, dict, matching realization (None if not found)
+    """
+    #check if point already exists
+    if self.restartData is not None:
+      # FIXME
```

Comment:
can you add more details?

Reply:
This is relocated from before; I don't know the intent of the commenter.  I don't see a problem with the line, although perhaps it was to find the `# OLD` line and restore it if something broke?  In this case, I am removing both.